pkg/controllers/federatedhpa/federatedhpa_controller.go

```diff
@@ -101,6 +101,27 @@ type timestampedRecommendation struct {
 	timestamp      time.Time
 }
 
+// hpaReqParams is parameters to get recommend metrics
+type hpaReqParams struct {
+	hpa            *autoscalingv1alpha1.FederatedHPA
```

Comment:
I prefer the name "fhpa" to distinguish it from native HPA.

Reply:
done